recipe/meta.yaml

```diff
@@ -20,6 +20,8 @@ requirements:
     - xz
   host:
     - gmp
+  run:
```

Comment:
@mariusvniekerk gmp should have run_exports https://github.com/conda-forge/gmp-feedstock/blob/master/recipe/meta.yaml#L15

Reply:
given that it failed at test time to load gmp it might not